engine/userguide/storagedriver/overlayfs-driver.md

```diff
@@ -82,22 +94,26 @@ Before following this procedure, you must first meet all the
     }
     ```
 
-    If you need to use the legacy `overlay` driver, specify it instead.
-
-    To use `overlay2` on CentOS (Docker CE only), you must also set the storage
-    option `overlay2.override_kernel_check`. In this case the `daemon.json`
-    would look like this:
+    > **Note**: RHEL and CentOS users on Docker EE 17.06
```

Comment:
Erm, sorry, just had a brain wave, but for the current Docker EE 17.06.2 release we don't support overlay2 (it's not in the the support matrix); the upcoming release has the kernel support check added, and will add overlay2 to the support matrix.

So, I don't think we should mention this, because this describes how to configure the daemon to use a graph driver that's not yet in the support matrix